dali/python/nvidia/dali/ops/__init__.py

```diff
@@ -57,7 +57,8 @@
 class _OpCounter(object):
     # pylint: disable=too-few-public-methods
     _lock = threading.Lock()
-    _op_count = count(0)
+    # start from something large to avoid confusion with (more common) per-pipeline numbering
```

Comment:
Note for reviewers: "avoid confusion" means only confusion while debugging. The renaming can deal with duplicate names.